imports/startup/server/email.js

```diff
@@ -1,3 +1,7 @@
 import { Meteor } from 'meteor/meteor';
 
-if (Meteor.isDevelopment) process.env.MAIL_URL = Meteor.settings.private.MAIL_URL;
+if (Meteor.settings.private && Meteor.settings.private.MAIL_URL) {
+  if (Meteor.isDevelopment) process.env.MAIL_URL = Meteor.settings.private.MAIL_URL;
+} else {
+  console.warn('Warning! You have not configured mail settings. Emails will not be sent.');
```

Comment:
Suggestion: Change `mail` and `Email` to `e-mail`. That way it's more consistent